hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1028,7 +1030,7 @@ public void testSetSnapshotSummary() throws Exception {
     }
     assertThat(JsonUtil.mapper().writeValueAsString(summary).length()).isLessThan(4000);
     Map<String, String> parameters = Maps.newHashMap();
-    ops.setSnapshotSummary(parameters, snapshot);
+    ops.getIcebergTableConverter().setSnapshotSummary(parameters, snapshot);
```

Comment:
Haven't you removed getIcebergTableConverter()? Does this code compile?

Reply:
Fixed.